hibernate-core/src/main/java/org/hibernate/query/internal/QueryOptionsImpl.java

```diff
@@ -160,6 +161,15 @@ public ResultListTransformer getResultListTransformer() {
 		return resultListTransformer;
 	}
 
+	@Override
+	public Boolean isDeDuplicationEnabled() {
+		return deDupEnabled;
+	}
+
+	public void setDeDuplicationEnabled(boolean enabled) {
+		this.deDupEnabled = enabled;
+	}
+
```

Comment:
who is calling this method?

Reply:
At the moment, no one; so we could drop it.  

I added that as part of initial work for supporting https://hibernate.atlassian.net/browse/HHH-15149.  But decided to delay that work until after 6.0 Final to better percolate.